documentation/modules/ROOT/pages/transformations/timezone-converter.adoc

```diff
@@ -0,0 +1,210 @@
+:page-aliases: configuration/timezone-converter.adoc
+[id="timezone-converter"]
+= Timezone Converter
+
+:toc:
+:toc-placement: macro
+:linkattrs:
+:icons: font
+:source-highlighter: highlight.js
+
+toc::[]
+
+[[timezone-converter-introduction]]
+
+When {prodname} emits event records, the timezone values for timestamp fields in the record can vary depending on the type and configuration of the data source.
+To maintain data consistency and precision within data processing pipelines and applications, you can use the `Timezone Converter` SMT to convert the time zone values of timestamp fields so that they represent data consistently.
+It is specifically helpful when you need to ensure that timestamps in your event data are consistent in timezone representation.
```

Comment:
```suggestion
To maintain data consistency and precision within data processing pipelines and applications, you can use the `Timezone Converter` SMT to ensure that event records use a consistent timezone to represent timestamp data.
```

Reply:
These two sentences are partially redundant. Does this successfully combine the thoughts from both of them?